pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -205,6 +205,13 @@ func (c *Cluster) Start() error {
 
 	// configure the mgr modules
 	c.configureModules(daemonIDs)
+
+	// Fetch performance counters from the prometheus exporter as ceph-exporter is disabled for 4.13
+	monStore := config.GetMonStore(c.context, c.clusterInfo)
```

Comment:
Could you put this inside the method `configurePrometheusModule()` to be consistent with #564?

Reply:
I coulnd't find such function for 4.13. I only see `configureModules()`

Reply:
I see it changed in 4.14, how about in `enablePrometheusModule()` for 4.13?

Reply:
Done!